src/fsharp/TcGlobals.fs

```diff
@@ -1497,6 +1509,84 @@ type public TcGlobals(compilingFslib: bool, ilg:ILGlobals, fslibCcu: CcuThunk, d
 
   member __.CompilerGeneratedAttribute = mkCompilerGeneratedAttribute ()
 
+  /// Find an FSharp.Core LaguagePrimitives dynamic function that corresponds to a trait witness, e.g.
+  /// AdditionDynamic for op_Addition.  Also work out the type instantiation of the dynamic function.
+  member __.makeBuiltInWitnessInfo (t: TraitConstraintInfo) =
```

Comment:
nit: Just a question, while other members in TcGlobals have Pascal-cased names, this one and the one below are camel-cased. What is the reasoning behind that? Just curious.

Reply:
Yes it's inconsistent, there were other functions on TcGlobals that used that casing.